release-notes/DataAggregator/DataAggregator_1.0.0.md

```diff
@@ -0,0 +1,358 @@
+---
+uid: DataAggregator_1.0.0
+---
+
+# Data Aggregator 1.0.0
+
+## New features
+
+#### New Data Aggregator DxM [ID_34725] [ID_34825] [ID_34914] [ID_34986] [ID_35047] [ID_35072] [ID_35450] [ID_35495]
+
+A new Data Aggregator DxM is now available. It can be used to schedule GQI queries to run periodically at fixed times, dates, or intervals. It can connect to multiple DataMiner Systems and combine the results of the GQI queries executed per DMS into one result. This result can then be exported to a CSV file or made available over a WebSocket connection.
+
+##### Installation and setup
+
+1. Make sure [DataMiner Cloud Pack](xref:CloudPackages) 2.8.4 or higher is installed on the server where you want to install Data Aggregator.
+
+1. Make sure DataMiner 10.2.12 or higher is installed on the DataMiner Agents you want to use Data Aggregator with.
+
+1. Make sure CoreGateway 2.12.0 or higher (included in Cloud Pack 2.8.4) is installed on the DataMiner Agents you want to use Data Aggregator with.
+
+1. Download the installation package for the Data Aggregator DxM from the [Dojo Downloads page](https://community.dataminer.services/downloads/) and install it.
+
+1. Open the folder `C:\Program Files\Skyline Communications\DataMiner DataAggregator`.
+
+1. Create a new file named *appsettings.custom.json* in this folder.
+
+   This is the file in which you will be able to customize the default configuration from *appsettings.json*. The settings configured in *appsettings.json* will be merged together with the settings configured in *appsettings.custom.json* and these merged settings will be applied by Data Aggregator.
+
+   > [!NOTE]
+   > Do not make changes to the existing *appsettings.json* file, as this file get overwritten during updates.
+
+1. Configure the necessary settings. See [Settings](#settings).
+
+1. Restart the *DataAggregator* service (e.g. using Windows Task Manager).
+
+##### Settings
+
+These settings should be configured in the *appsettings.custom.json* file.
+
+**HTTP listening**
+
+Configure which URL or URLs Data Aggregator should listen to. To specify multiple URLs, use semicolons (";") as separators.
+
+For example:
+
+```json
+{
+  "Urls": "http://127.0.0.1:5000"
+}
+```
+
+> [!IMPORTANT]
+> Make sure the specified port is available and not used by any other process.
+
+> [!NOTE]
+>
+> - Once the Data Aggregator setup is complete, if you browse to the configured URL and port, a web UI is displayed where you can start the queries manually, cancel ongoing queries, and check information such as the time it takes to finish a query, the total number of rows, and the number of rows received per second.
+> - Using the REST API, you can also do certain actions like getting the status of the jobs or manually triggering a specific job. More information is available via the URL `[Your configured URL]/swagger/index.html`, e.g. `http://127.0.0.1:5000/swagger/index.html`.
+
```

Comment:
I think we should also add a warning here that if no firewall is in place, anyone can access this web UI and the REST API, the DataAggregator has no authentication.